python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -1453,13 +1453,13 @@ def setDefaults(self):
         self.assembleStaticSkyModel.statistic = "MEANCLIP"
         self.assembleStaticSkyModel.sigmaClip = 2.5
         self.assembleStaticSkyModel.clipIter = 3
-        self.assembleStaticSkyModel.calcErrorFromInputVariance = False
+        self.assembleStaticSkyModel.calcErrorFromInputVariance = True
         self.assembleStaticSkyModel.doWrite = False
         self.detect.doTempLocalBackground = False
         self.detect.reEstimateBackground = False
         self.detect.returnOriginalFootprints = False
         self.detect.thresholdPolarity = "both"
-        self.detect.thresholdValue = 5
+        self.detect.thresholdValue = 6
```

Comment:
I thought your examples with `thresholdValue = 7.5` looked convincing, but no objection to 6.